README.md

```diff
@@ -236,6 +236,7 @@ A future feature of Scribe is language practice within the app itself. Scribe pr
 <strong>2023</strong>
 
 - [Presentation slides](https://docs.google.com/presentation/d/1W4ZkGi9UDDiTxM_silEij0gTE8YEubluHxe78xoqEP0/edit?usp=sharing) for a talk at [Berlin Hack and Tell](https://berlinhackandtell.rocks/) ([Hack of the month winner 🏆](https://berlinhackandtell.rocks/2023-03-28-no87-moore-hacks))
+- [Blog post 1](https://saurabhjamadagni.hashnode.dev/nested-uitableviews-apples-built-in-viewcontrollers) for documenting GSoC '23 with Scribe
```

Comment:
:tada::raised_hands: 

IIRC what you shared before, there'll be more posts, right?

Quick nit - but one way that the posts could be added too:
```suggestion
- [Blog post](https://saurabhjamadagni.hashnode.dev/nested-uitableviews-apples-built-in-viewcontrollers) on {THE WORK I DID THAT THE POST GOES OVER} for [GSoC '23]({URL TO MAYBE PHABRICATOR/GSOC WHERE THE SCRIBE GSOC INITIATIVE IS DOCUMENTED})
```

More so thinking that this might provide a more specific synopsis to help folks who come across the `Featured by`

Reply:
Ya I think that that would be more appealing for people to click on :) Also, the featured by section starts with the newest elements, so this should go above the Berlin Hack and Tell slides 🙃

Reply:
Ahh I didn't know that for sure, but intuitively that makes sense :+1: 

I don't think the specific day would be necessary, but could it make sense to prefix each listing with the month at least?

---

If so, doesn't have to happen in this PR; can happen later. Just something I thought of.

Reply:
Ya I can do a commit with those later 🙃 Adding it to my tasks for this week :)

Reply:
Yeah that makes a lot of sense. I was skeptical about the content to add. Let me make the above changes. Here's a [link](https://www.mediawiki.org/wiki/Google_Summer_of_Code/2023#Accepted_projects) to Mediawiki accepted projects page where Scribe is mentioned. Does this sound like a good link to redirect to or is there anything else I should link to?

Reply:
That link makes sense to me, @SaurabhJamadagni! :) :) Thank you! 🙌

Reply:
Pushed the changes right now. Thanks for the suggestion @wkyoshida 😄

Reply:
Final question on this: looking again, should the link be titled [Scrbe development blog post](https://saurabhjamadagni.hashnode.dev/nested-uitableviews-apples-built-in-viewcontrollers) instead of just `Blog post`? Just jumped out at me as maybe being a bit better :)

Reply:
How about 'Scribe iOS development blog post on Nested UITableViews & Apple's built-in ViewControllers in app menu for GSoC '23'?

Reply:
Yes that'd be solid 😊